src/data/achievements.js

```diff
@@ -321,6 +326,15 @@ const achievements = [
       return addItemToInventory(state, itemsMap['gold-ingot'], 1, true)
     },
   }))(),
+
+  (() => ({
```

Comment:
The achievements are presented to the player in the order they are implemented in this array. Since paying off the loan is typically an early game task, let's move this to the fourth position in this array (under `harvest-crop`).